### PR TITLE
Content Enhancements - Second Batch

### PR DESCRIPTION
**Pull request by nasbench**
This PR fixes/enhances some old analytics as well as a couple of new one.

### Windows CertUtil Download With URL Argument

Screenshot 2024-12-08 011720

### Windows Impair Defenses Disable Auto Logger Session

image

### Windows Registry Dotnet ETW Disabled Via ENV Variable

Screenshot 2024-12-08 183020

### Disabling Remote User Account Control

Removes the `HKLM\\SOFTWARE` prefix to make it more generic for other variations such `HKEY_LOCAL_MACHINE`

### Enable RDP In Other Port Number

Removes the `HKLM\\SOFTWARE` prefix to make it more generic for other variations such `HKEY_LOCAL_MACHINE`

### LOLBAS With Network Traffic

Adds `pwsh.exe` since `powershell.exe` and `powershell_ise.exe` are already in there, as well as fixes some typos.

### Registry Keys Used For Persistence

Removes the `HKLM` prefix from `HKLM\\SOFTWARE\\Microsoft\\Netsh\\* 

### Windows Registry BootExecute Modification

Removes the `HKLM` prefix from `HKLM\\System\\CurrentControlSet\\Control\\Session`

### Windows SQL Spawning CertUtil

See https://github.com/splunk/security_content/pull/3224#issuecomment-2525271089 for details. 
TL;DR is that it enhances coverage by adding other arguments and focusing only on the required ones.

---

The rest is some generic typo fix for the word `official`

### CODE REVIEW

**Review by patel-bhavin — APPROVED** [review]

**Comment by patel-bhavin** [comment]
Looking good @nasbench!! Approved :catjam:
